### PR TITLE
Added Obsolete attribute to old methods

### PR DESCRIPTION
**Pull request by jboyer2012**
I added Obsolete to the Read...AsDT and Read...AsList methods in FileHelperEngine.

### CODE REVIEW

File: FileHelpers/Engines/FileHelperEngine.cs [hunk 1]
```diff
@@ -107,6 +107,7 @@ public T[] ReadFile(string fileName, int maxRecords)
         }
 
         /// <include file='FileHelperEngine.docs.xml' path='doc/ReadFile/*'/>
+        [Obsolete]
```
Comment (MarcosMeli) [inline FileHelpers/Engines/FileHelperEngine.cs]: Great !! Can you provide a description like [Obsolete("You must use ReadFile(..).ToList()")] and likewise for other methods ?

Thanks

**Comment by MarcosMeli** [comment]
Great !! Can you provide a description like [Obsolete("You must use ReadFile(..).ToList()")] and likewise for other methods ?

Thanks

**Comment by jboyer2012** [comment]
I updated the obsolete tags with the workaround comments and committed.

**Comment by MarcosMeli** [comment]
Merged :) I didn't saw the commit

THANKS !!

**Pull request merged**